### DISCUSSION TITLE
Add prune peer dependence option

### DISCUSSION THREAD

**Discussion by Qquanwei** [npm/feedback#399] [CLI] [answered]
The case is that using `npm link` with `webpack` can cause peer dependence conflicted.  To resolve this situation need uninstall peerdependence manually. So we might provide a option like `npm prune --peer` to automate uninstalled that's packages.

**Comment by MylesBorins** [comment, marked as answer]
@Qquanwei if you would like to make a suggest for a new feature to the npm CLI the next step is to open an issue over at https://github.com/npm/rfcs

Thanks for opening the discussion!
